http_builder.go

```diff
@@ -247,6 +248,13 @@ func (b *builder) UseApplicationXML() *builder {
 	return b
 }
 
+func (b *builder) DumbOutRequest(requestString *string) *builder {
```

Comment:
will be called before or after callAPI?

Reply:
Before calling the API of course. This function will be called when we build a new request.
Example:
```gotemplate
var requestString *string
j.cli.Builder("").
    SetHeader("Content-Type", "application/json").
    SetHeader("signature", cfg.Signature).
    Post().
    DumbOutRequest(requestString).
    SetBody(request).Call(ctx, response)

// expected requestString ="GET / HTTP/1.1 Host: example.com User-Agent: Go-http-client/1.1 Test-Header: test-header-value Accept-Encoding: gzip"
```